sharding/shard.go

```diff
@@ -175,11 +181,13 @@ func (s *Shard) SaveHeader(header *CollationHeader) error {
 
 // SaveBody adds the collation body to the shardDB and sets availability.
 func (s *Shard) SaveBody(body []byte) error {
-	// TODO: check if body is empty and throw error.
-	// TODO: dependent on blob serialization.
-	// chunkRoot := getChunkRoot(body) using the blob algorithm utils.
-	// right now we will just take the raw keccak256 of the body until #92 is merged.
-	chunkRoot := common.BytesToHash(body)
+	// check if body is empty and throw error.
+	if body == nil {
```

Comment:
This only checks if the body is nil, did you also want to check len(body) == 0? I think that is the true condition for an empty body.